### PR TITLE
Reviews Functionality Completed

### PR DESCRIPTION
**Pull request by alyssaperkins**
updated functionalities:
- changes all home router names to profile
- added current user's community rating to profile page
- in past donations, there is a toggled 'Leave a Review' button that lets a user leave a review for a past donation
- note: donators can review donatees and donatees can review the donator to affect their community rating
- once a review is sent, the button toggles to "review completed" and is disabled
- reviews and rating data is sent to the reviews table in Supabase
- added watch method to dynamically update the ratings on pantry page donation cards

Current four cypress tests passed 4/4:
image

### CODE REVIEW

**Comment by alyssaperkins** [comment]
BUG: Leave a Review is not toggling - find out if this issue is on my end or from someone else

**Comment by alyssaperkins** [comment]
Fixed TC failures.
image

File: Client/src/pages/sign_in.vue [hunk 1]
```diff
@@ -1,68 +1,83 @@
 <template>
-  <q-page class="column items-center">
+  <q-page class="self-center">
```
Comment (Alec8888) [inline Client/src/pages/sign_in.vue:2]: I think it looks better with "items-center" instead of "self-center"

Comment (alyssaperkins) [reply Client/src/pages/sign_in.vue]: this is done

**Review by Alec8888 — APPROVED** [review]
Display of user rating is working. Approved.

**Pull request merged**